core/src/main/java/com/linecorp/armeria/server/VirtualHost.java

```diff
@@ -181,6 +184,12 @@ static String normalizeDefaultHostname(String defaultHostname) {
         return Ascii.toLowerCase(defaultHostname);
     }
 
+    static String normalizeContextPath(String contextPath) {
+        requireNonNull(contextPath, "contextPath");
+        // TODO Implement this
```

Comment:
I think we can just do something like the following.
```
RouteUtil.ensureAbsolutePath(contextPath, "contextPath");
```

We probably don't even need to define a dedicated method for this

Reply:
aha, thank you. let me address it.